decentraland/unity-renderer
# refactor: remove biw controller monobehaviour

## What does this PR change?

This PR removes completely the monobehaviour from the builder in world controllers. 

The controllers have been refactored to be interfaces so we can have an easy unit testing development

Now it will use the FeatureController to control all the monobehaviour callbacks

## How to test the changes?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:refactor/biw-2
2. Enter builder in world
3. Test that everything works as expected

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
